core.js

```diff
@@ -6,6 +6,7 @@ const names = require("./names.js");
 const feats = require("./feats.js");
 const armors = require("./armor");
 const fs = require("fs");
+const background = require("./background.js");
```

Comment:
It seemed like this was needed? but I have no idea

Reply:
Yep, so when you want to make access of another files functions you need to include this statement in the file calling the function so it knows where the remote function is stored.